spec/std/http/cookie_spec.cr

```diff
@@ -81,18 +81,50 @@ module HTTP
         cookie = parse_set_cookie("key=value; Secure")
         cookie.name.should eq("key")
         cookie.value.should eq("value")
-        cookie.secure.should eq(true)
+        cookie.secure.should be_true
         cookie.to_set_cookie_header.should eq("key=value; path=/; Secure")
       end
 
       it "parses HttpOnly" do
         cookie = parse_set_cookie("key=value; HttpOnly")
         cookie.name.should eq("key")
         cookie.value.should eq("value")
-        cookie.http_only.should eq(true)
+        cookie.http_only.should be_true
         cookie.to_set_cookie_header.should eq("key=value; path=/; HttpOnly")
       end
 
+      describe "SameSite" do
+        context "Lax" do
+          it "parses samesite" do
+            cookie = parse_set_cookie("key=value; SameSite=Lax")
+            cookie.name.should eq "key"
+            cookie.value.should eq "value"
+            cookie.samesite.should eq HTTP::Cookie::SameSite::Lax
```

Comment:
```suggestion
            cookie.samesite.should eq :lax
```

Reply:
This is not an improvement. I'd accept both variants, but changing it has no benefit.